Dockerfile

```diff
@@ -49,6 +48,7 @@ RUN curl "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terrafor
 ENV PLUGIN_OPENSTACK=0.2.2
```

Comment:
please group versions at the beginning of the file... I accidentally merged this as it is now